dev/NavigationView/NavigationView.cpp

```diff
@@ -3151,6 +3152,38 @@ void NavigationView::UpdateListViewItemSource()
     }
 }
 
+void NavigationView::UpdateSelectionForMenuItems()
```

Comment:
Should we just listen on IsSelected property change on the items?

Reply:
Nope. It's provided by parent of ListViewItem.